### ISSUE TITLE
CloudTracker does not work on EC2

### ISSUE THREAD

**Issue by bbbales2** [StochSS/stochss#493]
I opened up a pull request for us to work on this: https://github.com/StochSS/stochss/tree/fix_rerun

There were a couple small issues. Of note, common/helper.py 420-1 had some very strange code.

Currently you should be able to run a default stochastic well mixed model, delete the output, click rerun. The job will run, but the output will fail to download. The issue is that outputURL is not being updated to the new outputURL. I can see in the celery logs that the job reruns properly and spits out a new URL (which I can manually download with my browser), but that new URL never makes it's way back to the DB on the client.

**Comment by bbbales2** [comment]
Update: I did a test. Look in tasks.py:724. For the first run, it correctly detects that the job is new.

For the rerun, it still thinks the job is new, and runs track_input again (I added some printfs and saw this in the celery logs). The condition here revolves around the existence/non-existence of a Manifest file (look in stochss/app/lib/cloudtracker.py for the code).

I remember Melissa told me the Manifests were stored in a secondary bucket. I don't think they're in the main StochSS output one.
